sections/06_discussion.md

```diff
@@ -25,46 +25,294 @@ human-level performance is irrelevant.*
 
 ### Interpretation
```

Comment:
The original partial draft included `tag:Alipanahi2015_predicting`.  Is that redundant now in this much more detailed discussion or should it be included?